components/enrichers/codeowners/main.go

```diff
@@ -81,10 +81,13 @@ func run() error {
 			enrichedIssues = append(enrichedIssues, eI)
 		}
 
-		return enrichers.WriteData(&apiv1.EnrichedLaunchToolResponse{
+		err := enrichers.WriteData(&apiv1.EnrichedLaunchToolResponse{
```

Comment:
how is this different from before? it still returns either an err or nil

Reply:
ha, I'm glad you asked! It's true, it still either returns `err` or `nil`.

But: It's inside a loop. The `return` always exits that loop _on the first iteration_.
The enricher never got to the second or third `LaunchToolResponse` that it was meant to parse.